tests/tools.vitruv.applications.pcmumlclass.tests/src/tools/vitruv/applications/pcmumlclass/tests/InterfaceConceptTest.xtend

```diff
@@ -1,113 +1,187 @@
 package tools.vitruv.applications.pcmumlclass.tests
 
-import org.eclipse.emf.ecore.util.EcoreUtil
 import org.eclipse.uml2.uml.Interface
+import org.junit.jupiter.api.Test
 import org.palladiosimulator.pcm.repository.OperationInterface
-import org.palladiosimulator.pcm.repository.Repository
 import org.palladiosimulator.pcm.repository.RepositoryFactory
-import tools.vitruv.applications.pcmumlclass.TagLiterals
-import org.junit.jupiter.api.Test
-
 import static org.junit.jupiter.api.Assertions.assertNotNull
 import static org.junit.jupiter.api.Assertions.assertTrue
-import static org.junit.jupiter.api.Assertions.assertFalse
-import java.nio.file.Path
+import static org.junit.jupiter.api.Assertions.assertEquals
+import static tools.vitruv.applications.pcmumlclass.tests.PcmUmlElementEqualityValidation.*
+import static extension tools.vitruv.applications.pcmumlclass.tests.PcmQueryUtil.*
+import static extension tools.vitruv.applications.testutility.uml.UmlQueryUtil.*
 
 /**
  *  This test class tests the reactions and routines are supposed to synchronize a pcm::OperationInterface
  * with its corresponding uml::Interface (in the contracts uml::Package corresponding to a pcm::Repository).
  * <br><br>
  * Related files: PcmInterface.reactions, UmlInterface.reactions, UmlInterfaceGeneralization.reactions
  */
-class InterfaceConceptTest extends PcmUmlClassApplicationTest {
+class InterfaceConceptTest extends ViewBasedPcmUmlClassApplicationTest {
 
 	static val TEST_INTERFACE_NAME = "TestInterface"
+	static val NEW_TEST_INTERFACE_NAME = "NewTestInterface"
+	static val PACKAGE_NAME = "rootpackage"
+	static val PACKAGE_NAME_FIRST_UPPER = PACKAGE_NAME.toFirstUpper
 
 	def checkInterfaceConcept(
 		OperationInterface pcmInterface,
 		Interface umlInterface
 	) {
 		assertNotNull(pcmInterface)
 		assertNotNull(umlInterface)
-		assertTrue(corresponds(pcmInterface, umlInterface, TagLiterals.INTERFACE_TO_INTERFACE))
 		assertTrue(pcmInterface.entityName == umlInterface.name)
-		// should be contained in corresponding repository and contracts package respectively
-		assertTrue(
-			corresponds(pcmInterface.repository__Interface, umlInterface.package,
-				TagLiterals.REPOSITORY_TO_CONTRACTS_PACKAGE))
-		// parent interfaces should correspond
-		val umlParentCorrespondences = pcmInterface.parentInterfaces__Interface.map [ pcmParent |
-			getCorrespondingEObjects(pcmParent, Interface).head
-		].toList
-		assertFalse(umlParentCorrespondences.contains(null))
-		assertFalse(
-			umlParentCorrespondences.map [ umlParent |
-				umlInterface.generalizations.exists[gen|EcoreUtil.equals(gen.general, umlParent)]
-			].exists[it == false]
-		)
+	// should be contained in corresponding repository and contracts package respectively
+	// parent interfaces should correspond
```

Comment:
Weird location for the comment. Why not add it as proper JavaDoc to the method?

Reply:
Forgot to delete legacy method, deleted in https://github.com/vitruv-tools/Vitruv-CaseStudies/pull/214/commits/f3388bcf3d8c6bcc02f139a7ed61df489eb345d4